gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -51,7 +52,18 @@ public async Task ShouldSendScriptForEvaluationAndReturnCorrectResult(string req
                 Assert.Equal(expectedResponse, response);
             }
         }
-
+        
+        [Fact]
+        public async Task ShouldReturnResponseAttributes()
+        {
+            var gremlinServer = new GremlinServer(TestHost, TestPort);
+            using (var gremlinClient = new GremlinClient(gremlinServer))
+            {
+                var requestMsg = _requestMessageProvider.GetDummyMessage();
```

Comment:
Please separate the three testing phases _act_, _arrange_, and _assert_ with empty lines between them to make it more clear which part of the test belongs to which phase and to stay consistent with the other tests. (See for example [this blog post](http://defragdev.com/blog/?p=783) that explains this structure and the phases.)

Reply:
Fixed